components/HelpModal.go

```diff
@@ -0,0 +1,77 @@
+package components
+
+import (
+	"github.com/gdamore/tcell/v2"
+	app "github.com/jorgerojas26/lazysql/app"
+	"github.com/jorgerojas26/lazysql/commands"
+	. "github.com/jorgerojas26/lazysql/keymap"
+	"github.com/rivo/tview"
+)
```

Comment:
I would expect a consistent import style with files available in the repository

```suggestion
import (
	"github.com/gdamore/tcell/v2"
	"github.com/rivo/tview"

	app "github.com/jorgerojas26/lazysql/app"
	"github.com/jorgerojas26/lazysql/commands"
	. "github.com/jorgerojas26/lazysql/keymap"
)
```